### PR TITLE
Capitalize the 'e' in "experimental"

### PR DESCRIPTION
**Pull request by PankajBhojwani**
## Summary of the Pull Request
Updating the resource strings to be in line with the mocks (capitalizing the "e", removing the unnecessary experimental tag in the dropdown)

## PR Checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated
   - If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
- [ ] Schema updated (if necessary)

### CODE REVIEW

**Review by lhecker — APPROVED** [review]

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -737,7 +737,7 @@
     <value>Command Palette</value>
   </data>
   <data name="AIChatMenuItem" xml:space="preserve">
-    <value>Terminal Chat (experimental)</value>
+    <value>Terminal Chat</value>
```
Comment (adrastogi) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:740]: >Terminal Chat [](http://example.com/codeflow?start=11&length=13)

Can you remind me the rationale for removing it from the dropdown? Was it to save the horizontal real estate?

**Review by adrastogi — APPROVED** [review]
:shipit:

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw (hunk 1 above)
Comment (PankajBhojwani) [reply src/cascadia/TerminalApp/Resources/en-US/Resources.resw:740]: From @nguyen-dows:

> I personally don’t think that “Experimental” should be marked on dropdown menu items especially when you’ll already see “Terminal Chat (Experimental)” after opening up the chat UI

Comment (nguyen-dows) [reply src/cascadia/TerminalApp/Resources/en-US/Resources.resw:740]: ^ Horizontal real estate was also a factor in this decision in excluding that word :)

Comment (adrastogi) [reply src/cascadia/TerminalApp/Resources/en-US/Resources.resw:740]: Makes sense- thanks, guys!

**Pull request merged**